### PR TITLE
fix: mobile project settings network labels

### PR DESCRIPTION
**Pull request by marandaneto**
## Problem

follow up https://github.com/PostHog/posthog/pull/26289

## Changes

👉 _Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review._

### CODE REVIEW

File: frontend/src/scenes/settings/environment/SessionRecordingIngestionSettings.tsx [hunk 1]
```diff
@@ -252,7 +252,7 @@ function UrlConfigSection({
             </div>
             <p>{description}</p>
 
-            <p>`${title} is only available for JavaScript Web.`</p>
+            <p>{title} is only available for JavaScript Web.</p>
```
Comment (marandaneto) [inline frontend/src/scenes/settings/environment/SessionRecordingIngestionSettings.tsx:255]: that's not TS :D

File: frontend/src/scenes/settings/environment/SessionRecordingSettings.tsx [hunk 2]
```diff
@@ -115,6 +115,13 @@ export function NetworkCaptureSettings(): JSX.Element {
                 This setting controls if performance and network information will be captured alongside recordings. The
                 network requests and timings will be shown in the recording player to help you debug any issues.
             </p>
+            <p>
```
Comment (marandaneto) [inline frontend/src/scenes/settings/environment/SessionRecordingSettings.tsx:118]: this is now after the desc. and before the toggle as any other.

File: frontend/src/scenes/settings/environment/SessionRecordingSettings.tsx [hunk 3]
```diff
@@ -133,13 +140,7 @@ export function NetworkCaptureSettings(): JSX.Element {
...
                     </Link>
                 </p>
-                <p>
-                    Network capture is also available for{' '}
-                    <Link to="https://posthog.com/docs/session-replay/network-recording" target="_blank">
-                        Mobile session replay
-                    </Link>{' '}
-                    , where they can be configured directly in code.
-                </p>
+                <p>Capture headers and body are only available for JavaScript Web.</p>
```
Comment (marandaneto) [inline frontend/src/scenes/settings/environment/SessionRecordingSettings.tsx:143]: so people don't try to find those on mobile via code

**Review by ioannisj — APPROVED** [review]
👍

**Pull request merged**